packages/astro/src/@types/astro.ts

```diff
@@ -895,11 +896,40 @@ export interface AstroUserConfig {
 		astroFlavoredMarkdown?: boolean;
 	};
 
-	/**
-	 * @hidden
-	 * Turn on experimental support for the new error overlay component.
+		/**
+	 * @docs
+	 * @kind heading
+	 * @name Experimental Flags
+	 * @description
+	 * Astro offers experimental flags to give users early access to new features.
+	 * These flags are not guaranteed to be stable.
 	 */
-	experimentalErrorOverlay?: boolean;
+	experimental?: {
+		/**
+			* @hidden
+			* Turn on experimental support for the new error overlay component.
+			*/
+		errorOverlay?: boolean;
+		/**
+		 * @docs
+		 * @name experimental.prerender
+		 * @type {boolean}
+		 * @default `false`
+		 * @version 1.7.0
+		 * @description
+		 * Enable experimental support for Prerendered pages when generating a server.
+		 *
+		 * To enable this feature, set `experimental.prerender` to `true` in your Astro config:
+		 *
+		 * ```js
+		 * {
+		 * 	experimental: {
+		 *		prerender: true,
+		 * 	},
+		 * }
```

Comment:
Think we probably need the closing backticks or we’ll hit a render error when pulling this into docs.
```suggestion
		 * }
		 * ```
```